api_test.py

```
@@ -0,0 +1,43 @@
+
+#, "--ignore-certificate-errors"
+
+payload = {
+    "task": "Naviagte to http://192.34.60.201/auth/login. Creds needed for login are: username - my@company.com, pasword - peter.",
+    "add_infos": "In the final result, include the url of the page the task finished at",
+    "url": "http://192.34.60.201/auth/login",
+    "gbc": True,
+    "upload_file_path:": None
+}
+
+response = requests.post(url, json=payload)
```

Ruff (0.8.2)

Probable use of `requests` call without timeout

(S113)